src/AppInstallerCommonCore/AppInstallerTelemetry.cpp

```diff
@@ -91,16 +109,54 @@ namespace AppInstaller::Logging
         m_isRuntimeEnabled = true;
     }
 
+    void TelemetryTraceLogger::SetTelemetryCorelationJson(const std::wstring_view& jsonStr_view) noexcept
+    {
+        // Check if passed in string is a valid Json formatted before returning the value
+        // If invalid, return empty Json
+        Json::CharReaderBuilder jsonBuilder;
+        Json::CharReader* jsonReader = jsonBuilder.newCharReader();
+        std::unique_ptr<Json::Value> pJsonValue = std::make_unique<Json::Value>();
+        std::string errors;
+        std::wstring jsonStrW{ jsonStr_view };
+        std::string jsonStr = ConvertToUTF8(jsonStrW.c_str());
+
+        bool result = jsonReader->parse(jsonStr.c_str(),
+            jsonStr.c_str() + jsonStr.size(),
+            pJsonValue.get(),
+            &errors);
+
+        delete jsonReader;
+
+        if (result)
+        {
+            m_telemetryCorelationJsonW = jsonStrW;
+            AICLI_LOG(Core, Info, << "Passed in Corelation Json is valid: " << jsonStr);
+        }
+        else
+        {
+            AICLI_LOG(Core, Info, << "Passed in Corelation Json is invalid: " << jsonStr << "; Error: " << errors);
+            m_telemetryCorelationJsonW = L"{}";
```

Comment:
The default value should already be this; plus we should make this an error, not a hidden swallow.